Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14
+FROM iron/node
```

Comment:
This looks real interesting. I will put this on the backburner and try it out. thanks! it looks like iron does not make the version of node i need at the moment :(